docs/pages/workflow/customizing.md

```diff
@@ -49,6 +49,37 @@ If you've decided that you want to roll your app back to being fully managed (no
 
 Once you have customized the native code in your project, you can use the [`expo-dev-client`](/development/introduction) package to create a development build and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a development build for your managed or bare workflow by following [our guide](/development/getting-started).
 
+### Integrating project-specific, symlinked native modules
+
+In some cases you may want to add a custom native module specific to you project. Ideally this custom native module should live within your project, however, if you are using the managed workflow you cannot safely add the project-specific module's native code in the `ios` or `android` directories. Instead the solution is to house any project-specific native module within a local `libs` folder and then symlink this module to your project. This can be achieved as follows:
+
+- First of all create a folder to house your modules under e.g. `mkdir libs`
```

Comment:
I’d rather we encourage the use of yarn workspaces (linking to our monorepo guide) and Expo Modules Core / Expo Autolinking — these are easier to scale and require less code.

Long term we’d like to scale Expo Autolinking to just detect swift and kotlin files in the project automatically. This would then effectively be the recommended easiest way to add managed native code. 

Guides regarding bob builder should probably live in the bob builder guide (which we could link to).

Reply:
That definitely makes a lot more sense for managed projects and thanks for the feedback @EvanBacon and @barthap 😃